scripts/pi-hole/php/savesettings.php

```diff
@@ -535,12 +555,6 @@ function readAdlists()
 
 					$domain = $_POST["domain"];
 
-					// Validate Domain name
-					if(!validDomain($domain))
-					{
-						$error .= "Domain name ".htmlspecialchars($domain)." is invalid!<br>";
-					}
```

Comment:
Why is this section removed?

Reply:
It wasn't meant to be, I thought it just moved.